sv-SE.lproj/Menu.strings

```diff
@@ -0,0 +1,258 @@
+"Check Spelling While Typing" = "Kontrollera stavning under skrift";
+"Check Grammar With Spelling" = "Kontrollera grammatik under skrift";
+"Correct Spelling Automatically" = "Korrigera stavning automatiskt";
+"Substitutions" = "Substitutioner"; /* Context? */
+"Smart Quotes" = "Smarta citattecken";
+"Smart Dashes" = "Smarta streck";
+"Text Replacement" = "Textersättning"; /* Context? */
+"Speech" = "Tal";
+"Start Speaking" = "Börja tala";
+"Stop Speaking" = "Sluta tala";
+
+/* Paragraph Menu */
+"Paragraph" = "Stycke";
+"Heading 1" = "Rubrik 1";
+"Heading 2" = "Rubrik 2";
+"Heading 3" = "Rubrik 3";
+"Heading 4" = "Rubrik 4";
+"Heading 5" = "Rubrik 5";
+"Heading 6" = "Rubrik 6";
+"Increase Heading Level" = "Öka rubriknivå";
+"Decrease Heading Level" = "Minska rubriknivå";
+"Table" = "Tabell";
+"Code Fences" = "Kodblock";
+"Math Block" = "Matteblock";
+"Quote" = "Citat";
+"Ordered List" = "Ordnad lista";
+"Unordered List" = "Oordnad lista";
+"Task List" = "Checklista";
+"Task Status" = "Status för checklista";
+"Toggle Task Status" = "Växla markering";
+"Mark as Complete" = "Markera";
+"Mark as Incomplete" = "Avmarkera";
+"List Indentation" = "Listans indrag";
+"Indent" = "Öka indrag";
+"Outdent" = "Minska indrag";
+"Link Reference" = "Länkreferens";
+"Footnotes" = "Fotnoter";
+"Horizontal Line" = "Horisontell linje";
+"Table of Contents" = "Innehållsförteckning";
+"YAML Front Matter" = "YAML-metadata";
+
+/* Format Menu */
+"Format" = "Format";
+"Strong" = "Fetstil";
+"Emphasis" = "Kursiv";
+"Underline" = "Understruken";
+"Code" = "Kod";
+"Inline Math" = "Inline matematik";
+"Strike" = "Genomstruken";
+"Highlight" = "Markera";
+"Superscript" = "Upphöjd";
+"Subscript" = "Nedsänkt";
+"Comment" = "Kommentar";
+"Hyperlink" = "Hyperlänk";
+"Image" = "Bild";
+"Clear Format" = "Rensa formattering";
+
+/* View Menu */
+"View" = "Visa";
+"New Tab" = "Ny flik";
+"Source Code Mode" = "Källkodsläge";
+"Focus Mode" = "Fokusläge";
+"Typewriter Mode" = "Skrivmaskinsläge";
+"Toggle Sidebar" = "Växla sidomeny";
+"Outline" = "Översikt";
+"Articles" = "Artiklar";
+"File Tree" = "Filträd";
+"Always on Top" = "Alltid överst";
+"Full Screen" = "Fullskärm";
+"Show Status Bar" = "Visa statusraden";
+"Toggle Fullscreen" = "Växla Fullskärm";
+"Actual Size" = "Verklig storlek";
+"Zoom In" = "Zooma in";
+"Zoom Out" = "Zooma ut";
+"Switch Between Opened Documents" = "Växla mellan öppna dokument";
+"Toggle DevTools" = "Växla utvecklingsverktyg";
+
+
+/* Themes Menu */
+"Themes" = "Teman";
+
+/* Window Menu */
+"Window" = "Fönster";
+"Minimize" = "Minimera";
+"Zoom" = "Zooma";
+"Bring All to Front" = "Bring All to Front"; /* Context? */
+
+/* Help Menu */
+"Help" = "Hjälp";
+"Privacy Policy" = "Integritetspolicy";
+"Credits" = "Credits";
+"Change Log" = "Ändringslogg";
+"Website" = "Hemsida";
+"Feedback" = "Feedback";
+"About" = "Om";
+
+/* Touchbar */
+"Block Styles" = "Block Styles"; /* Context? */
+"Inline Styles" = "Inline Styles"; /* Context? */
```

Comment:
There a name of a button group (or toolbar group)

The first group are responsible for changing block styles (between paragraph, h1~h5, table, etc)
The send group are responsible for changing inline styles (between normal text, strong, emphasis, code, etc)